268 rta all docs (#3844)
timescale/docs: use-timescale/hyperfunctions/heartbeat-agg.md

Given a series of timestamped health checks, it can be tricky to determine the
overall health of a system over a given interval. PostgresQL provides window

> functions which can be used to get a sense of where unhealthy gaps are, but can

Reviewer: which you use to

be somewhat awkward to use efficiently. 

> This is one of the many cases where Timescale provides an efficient, simple solution for 

Reviewer: ```suggestion
This is one of the many cases where hyperfunctions provide an efficient, simple solution for 
```

 ("2012-03-25 03:00:51+00","2012-04-11 00:01:00+00")
```

> You can construct more elaborate queries, such as looking for the 5 units with the

Reviewer: ```suggestion
You can construct more elaborate queries. For example, to return the 5 units with the
```
